src/layouts/Settings/AnalyticsSettings.tsx

```diff
@@ -29,7 +29,11 @@ export const AnalyticsSettings = ({
       <VStack spacing="1.5rem" align="flex-start" w="50%">
         <FormControl isDisabled={isError}>
           <FormLabel>Facebook Pixel</FormLabel>
-          <Input w="100%" {...register("pixel")} />
+          <Input
+            w="100%"
+            type="number"
+            {...register("pixel", { valueAsNumber: true })}
+          />
```

Comment:
nit: we might want to validate this to be of length 15.

Reply:
sanity check: what happens to existing values that are of type string ah?